core/src/scene/styleContext.cpp

```diff
@@ -233,7 +233,6 @@ bool StyleContext::parseStyleResult(StyleParamKey _key, StyleParam::Value& _val)
             case StyleParamKey::color:
             case StyleParamKey::outline_color:
             case StyleParamKey::font_fill:
-            case StyleParamKey::font_stroke:
```

Comment:
[Documentation](https://github.com/tangrams/tangram-docs/blob/gh-pages/pages/draw.md#stroke), says `stroke` can be a `scalar` value, which will get mapped to the color of the stroke. But I am not sure how useful this is!